.travis.yml

```diff
@@ -8,6 +8,15 @@ cache:
         - node_modules
         - $TRAVIS_BUILD_DIR/target
 
+# $BOX_HOST_NAME and $BOX_PORT are also used by Selenium tests.
+env:
+  - BOX_HOST_NAME=foxbox BOX_PORT=3000
```

Comment:
Original reasoning behind that: custom host name is currently not supported for MacOS, so mac users will still be able to use ```npm test``` locally.